src/app/devices/deviceContent/components/deviceInterfaces/deviceInterfaces.tsx

```diff
@@ -104,8 +106,30 @@ export default class DeviceInterfaces extends React.Component<DeviceInterfacePro
                 >
                         {context.t(ResourceKeys.deviceInterfaces.command.configure)}
                 </ActionButton>
-                <Label> {context.t(ResourceKeys.deviceInterfaces.columns.displayName)}: {displayName}</Label>
-                <Label> {context.t(ResourceKeys.deviceInterfaces.columns.description)}: {description}</Label>
+                <MaskedCopyableTextFieldContainer
+                    ariaLabel={context.t(ResourceKeys.deviceInterfaces.columns.id)}
+                    label={context.t(ResourceKeys.deviceInterfaces.columns.id)}
+                    value={getInterfaceIdFromQueryString(this.props)}
+                    allowMask={false}
+                    t={context.t}
```

Comment:
why do we need to pass "t" ?

Reply:
wasn't going to change this one. but since the pr is paying off some debt, sure why not.